Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1694,6 +1768,779 @@ def is_ip_valid(s, accept_v6_ips=False):
         return True
 
 
+class Indicator(object):
+    """
+    interface class
+    """
+    @abstractmethod
+    def to_context(self):
+        pass
+
+
+class CommandResults:
+    """
+    CommandResults class - use to return results to warroom
+
+    :type outputs_prefix: ``str``
+    :param outputs_prefix: should be identical to the prefix in the yml contextPath in yml file. for example:
+            CortexXDR.Incident
+
+    :type outputs_key_field: ``str``
+    :param outputs_key_field: primary key field in the main object
```

Comment:
I think that we should mention that it is used for the DT, because the current explanation is a bit vague